spec/commands/list_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe "bundle list", :bundler => "2" do
+RSpec.describe "bundle list", :bundler => "3" do
```

Comment:
`bundle list` should be tested on bundler 2.

Reply:
Forget about it, this is correct. `bundle list` will be a new command in bundler 3. :+1: